msgcntr/messageforums-app/src/webapp/jsp/privateMsg/pvtMsgFolderAdd.jsp

```diff
@@ -30,7 +30,7 @@
 			   <h:messages styleClass="alertMessage" id="errorMessages" rendered="#{! empty facesContext.maximumSeverity}" />
 			 
  <h:panelGrid styleClass="jsfFormTable" columns="2">
-			  <h:panelGroup styleClass="shorttext required">
+			  <h:panelGroup styleClass="shorttext required form-control-label">
```

Comment:
form-control-label is from _sitemanage.scss, it is part of the site manage tool's styles, not the msgcntr styles. So, maybe you should try using bootstrap's form-label class instead?

Reply:
Hi Adrian, thanks for your suggestion! It seems like ```form-label``` does not work in this case, so I move ```form-control-label``` from ```_sitemanage.scss``` to ```tool.scss```. Would you mind taking a look at it?